packages/providers/src.ts/base-provider.ts

```diff
@@ -1875,3 +1869,43 @@ export class BaseProvider extends Provider implements EnsProvider {
         return this;
     }
 }
+
+
+// resolves network string to a hedera network name
+function resolveNetwork(net: Network | string | number | Promise<Network>) {
```

Comment:
I think that this should be part of the `DefaultHederaProvider` and here we should use the concrete `URL`s for the 2 endpoints (consensus + mirror)

Reply:
Will not resolve this for now, as there is no support for URLs in the `hashgraph` sdk